app/models/user.rb

```diff
@@ -140,6 +140,14 @@ def self.exclude_blockouts(start_at, end_at)
       .where(blockouts: { id: nil })
   end
 
+  def destroy
+    update_attributes(deactivated: true) unless deactivated
+  end
```

Comment:
Let's not implement this just yet. If we were building a full-on soft-delete implementation (including default scopes, etc) then I think this would make sense. We're kinda playing with the idea right now, though.

```suggestion
```